### ISSUE TITLE
"Invalid channel" error on Windows

### ISSUE THREAD

**Issue by farisissa8** [LedgerHQ/ledger-node-js-api#4]
I can't seem to get this to work on Windows (Node). The device is detected, but when I attempt to get the BTC public key I get an error "Invalid channel".

The code out of test/testBTC.js:
```
var ledger = require('ledgerco');

ledger.comm_node.create_async(0, true).then(function(comm) {

  var btc = new ledger.btc(comm);
  btc.getWalletPublicKey_async("44'/0'/0'/0").then(function(result) {
    console.log(result);
  }).fail(function(ex) {console.log(ex);});

}).fail(function(ex) {console.log(ex);});
```

The console shows:
```
=>01010500000016e040000011048000002c80000000800000000000000000000000000000000000
00000000000000000000000000000000000000000000000000
<=0105000000000040000011048000002c8000000080000000000000000000000000000000000000
00000000000000000000000000000000000000000000000000
Invalid channel
```
The same snippet works on Ubuntu. What am I missing?

**Comment by btchip** [comment]
If you still have Browser Support enabled on your device application, it looks like Windows loaded the wrong driver. You can try to delete all related USB drivers, remove the device, insert it again and let it reload the proper one.

**Comment by farisissa8** [comment]
Browser support is turned off. I tried removing all the drivers, but unfortunately there is no change. Windows re-installs the same two drivers which are: "USB Input Device", and "HID-compliant vendor-defined device". I also tried on a different machine with the same result.

One thing to note is that there are two "Invalid channel" errors, one that happens when Browser support is enabled (that logs "Invalid channel;" with a semi-colon), and this other one that I'm experiencing (without a semi-colon). I hope that helps.

**Comment by btchip** [comment]
Sorry, I meant that Browser Support needs to be enabled. The communication log is still pretty weird. Can you check if the U2F application works with a standard U2F backend, such as https://u2fdemo.appspot.com (or Google accounts, or Github) ?

**Comment by farisissa8** [comment]
Yes that works fine. The Ledger BTC chrome app works fine as well (when not in browser support mode of course), I thought I'd mention that.

**Comment by btchip** [comment]
ok, one more test. Can you check the BTC app with BitGo or the ETH app with MyEtherWallet ? both in Browser Mode. That would tell if the problem comes from the Node API mapping or something else.

**Comment by farisissa8** [comment]
Both work.

**Comment by btchip** [comment]
ok, so I guess I'll have to recheck the browser mapping, it might just be broken.

**Comment by btchip** [comment]
For reference this was fixed in 1.1.1
